### PR TITLE
Arachne 1882 front end for my data sources at central

### PR DESCRIPTION
**Pull request by johnSamilin**

### CODE REVIEW

File: src/const/dataSource.js [hunk 1]
```diff
@@ -224,6 +230,95 @@ const immutableAttributes = [
...
+function getDataSourceCreationFields(dbmsTypeList) {
+  return [
+    {
+      name: 'name',
+      className: 'col-xs-12',
+      InputComponent: {
+        component: FormInput,
+        props: {
+          mods: ['bordered'],
+          placeholder: 'Name of data source *',
```
Comment (pavgra) [inline src/const/dataSource.js]: isRequired

File: src/modules/DataCatalog/components/MyDatasources/components/Actions/presenter.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+function Actions(props) {
+  const classes = new BEMHelper('actions');
+  const {
+    openCreateModal,
+  } = props;
+
+  return (
+    <div {...classes()}>
+      {/*<Button mods={['rounded', 'submit']} onClick={openCreateModal}>Create Datsource</Button>*/}
```
Comment (pavgra) [inline src/modules/DataCatalog/components/MyDatasources/components/Actions/presenter.jsx]: ?

File: src/modules/DataCatalog/components/MyDatasources/container.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+export default class MyDatasourcesBuilder extends ContainerBuilder {
+  getComponent() {
+    return MyDatasources;
+  }  
+ 
+  mapStateToProps(state) {
+    const queryParams = get(state, 'routing.locationBeforeTransitions.query');
+
+    return {
+      filterFields: [], // selectors.getFilterList(state),
```
Comment (pavgra) [inline src/modules/DataCatalog/components/MyDatasources/container.js]: ?

File: src/modules/DataCatalog/components/MyDatasources/components/Actions/presenter.jsx (hunk 2 above)
Comment (johnSamilin) [reply src/modules/DataCatalog/components/MyDatasources/components/Actions/presenter.jsx]: that's not ready on backend yet

**Pull request merged**